.github/workflows/publish-1.0-docs.yml

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Based on Apache Arrow's java-nightly workflow
+# https://github.com/apache/arrow/blob/master/.github/workflows/java_nightly.yml
+name: Publish 1.0 documentation
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  publish-nightly:
+    name: Publish 1.0 docs
+    runs-on: ubuntu-20.04
+    if: github.repository == 'apache/incubator-pekko'
+    steps:
+      # TODO we will need to change to a 1.0 branch or a release tag
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6
+
+      - name: Install Graphviz
+        run: |-
+          sudo apt-get install graphviz
+
+      # TODO come up with a better way to control the version, possibly based on git tags
+      - name: Build Documentation
+        run: |-
+          sbt -Dpekko.genjavadoc.enabled=true "set ThisBuild / version := \"1.0.0\"; docs/paradox; unidoc"
+
+      # Create directory structure upfront since rsync does not create intermediate directories otherwise
+      - name: Create nightly directory structure
+        run: |-
+          mkdir -p target/nightly-docs/docs/pekko/1.0/
```

Comment:
I just noticed but this needs to be `1.0.0`, not `1.0`

Reply:
that's why I brought up the 2.6 example

Reply:
> that's why I brought up the Akka 2.6 example

Right but thats because Akka 2.6 is not the latest release branch, 2.8 is and for the 2.8 branch Akka does publish all patch versions as stated. In the case of Pekko 1.0 is our latest release branch

> If we insist on 1.0.0, we will probably need to discuss with infra team first to see if they are happy for us to use so much disk space. 1.0 strategy will use a lot less disk space than 1.0.x strategy.

We can but it shouldn't block us if we use the same strategy that Akka does, i.e. only do the patch doc publishing for the latest one (maybe 2) release branches.

Reply:
changed to 1.0.0